src/platform/plugins/shared/inspector/public/ui/inspector_view_chooser.tsx

```diff
@@ -69,57 +63,43 @@ export class InspectorViewChooser extends Component<Props, State> {
     );
   };
 
-  renderViewButton() {
-    return (
-      <EuiButtonEmpty
-        size="s"
-        iconType="arrowDown"
-        iconSide="right"
-        onClick={this.toggleSelector}
-        data-test-subj="inspectorViewChooser"
-      >
-        <FormattedMessage
-          id="inspector.view"
-          defaultMessage="View: {viewName}"
-          values={{ viewName: this.props.selectedView.title }}
-        />
-      </EuiButtonEmpty>
-    );
-  }
-
   renderSingleView() {
     return (
-      <EuiToolTip position="bottom" content={this.props.selectedView.help}>
```

Comment:
`this.props.selectedView.help` is not longer displayed anywhere on the page. Can we keep this? Its useful information.